usecases/schema/handler.go

```diff
@@ -170,33 +172,35 @@ func NewHandler(
 	return handler, nil
 }
 
-// GetSchema retrieves a locally cached copy of the schema
-func (h *Handler) GetSchema(principal *models.Principal) (schema.Schema, error) {
-	err := h.Authorizer.Authorize(principal, authorization.READ, authorization.CollectionsMetadata()...)
-	if err != nil {
-		return schema.Schema{}, err
-	}
-
-	return h.getSchema(), nil
```

Comment:
(note: let's check if we can also delete this function)

Reply:
it's used in `GetConsistentSchema()`